test/controllers/customers_controller_test.rb

```diff
@@ -0,0 +1,48 @@
+require "test_helper"
+
+class CustomersControllerTest < ActionDispatch::IntegrationTest
+  setup do
+    @customer = customers(:one)
+  end
+
+  test "should get index" do
+    get customers_url
+    assert_response :success
+  end
+
+  test "should get new" do
+    get new_customer_url
+    assert_response :success
+  end
+
+  test "should create customer" do
+    assert_difference("Customer.count") do
+      post customers_url, params: { customer: { email: @customer.email, first_name: @customer.first_name, last_name: @customer.last_name, phone: @customer.phone } }
```

Comment:
This can work with the current setup since customers don't have any uniqueness constraints but generally they will. Tests to create new customers will usually use fresh data instead of the attributes of the `customers(:one)` already in the DB so they're easier to tell apart and write more tests for. :)